iOSNativeLoginTemplate/iOSNativeLoginTemplate/NativeLoginView.swift

```diff
@@ -155,8 +268,128 @@ struct NativeLoginView: View {
                 SalesforceManager.shared.nativeLoginManager().fallbackToWebAuthentication()
             }
         }.background(Gradient(colors: [.blue, .cyan, .green]).opacity(0.6))
-            .blur(radius: self.isAuthenticating ? 2.0 : 0.0)
+            .blur(radius: isAuthenticating ? 2.0 : 0.0)
     }
+    
+    ///
+    /// Submits the OTP delivery request when the request button is tapped.  This submits a request to the
+    /// `init/passwordless/login`endpoint and opens the submit OTP verification view on success.
+    ///
+    private func onRequestOtpTapped() {
+        // Reset the error message if needed.
+        errorMessage = ""
+        
+        // Show the progress indicator.
+        isAuthenticating = true
+        
+        // Execute for a new reCAPTCHA token.
+        reCaptchaClientObservable.reCaptchaClient?.execute(withAction: .login) {reCaptchaExecuteResult, error in
+            
+            // Guard for the new reCAPTCHA token.
+            guard let reCaptchaToken = reCaptchaExecuteResult else {
+                print("Could not obtain a reCAPTCHA token due to error with description '\(error?.localizedDescription ?? "(A description wasn't provided.)")'.")
```

Comment:
for this and other print statements, could SalesforceLogger be used instead?

Reply:
Good suggestion.  Done!  I also cleaned up print use from `dev` as well.